sdk/core/azure-core/src/main/java/com/azure/core/cryptography/package-info.java

```diff
@@ -2,6 +2,26 @@
 // Licensed under the MIT License.
 
 /**
- * Package containing core cryptography interfaces.
+ * <p>This package contains cryptography interfaces for Azure SDK client libraries. These interfaces allow client
+ * libraries to perform cryptographic operations using asymmetric and symmetric keys, such as encrypting, decrypting,
+ * signing, verifying, wrapping, and unwrapping keys. The package also provides classes that can resolve key
```

Comment:
```suggestion
 * libraries to perform cryptographic operations using asymmetric and symmetric keys, such as encrypting (wrapping), decrypting (unwrapping),
 * signing, and verifying. The package also provides classes that can resolve key
```

- maybe? Just making it clear that those are the same operation.